.cog/schemas/composable/bigquery/dataquery.cue

```diff
@@ -0,0 +1,76 @@
+package bigquery
+
+import (
+	"github.com/grafana/grafana/packages/grafana-schema/src/common"
+)
+
+// Manually converted from https://github.com/grafana/google-bigquery-datasource/blob/18680e42ba557791d109c7c540c2c3f2647592f0/src/types.ts#L75
+Dataquery: {
+    common.DataQuery
+
+    dataset?: string
+    table?: string
+    project?: string
+
+    format: #QueryFormat
+    rawQuery?: bool
+    rawSql: string
+    location?: string
+
+    partitioned?: bool
+    partitionedField?: string
+    convertToUTC?: bool
+    sharded?: bool
+    queryPriority?: #QueryPriority
+    timeShift?: string
+    editorMode?: #EditorMode
+    sql?: #SQLExpression
+
+    #QueryFormat: 0 | 1 @cog(kind="enum", memberNames="Timeseries|Table")
+    #QueryPriority: "INTERACTIVE" | "BATCH" @cog(kind="enum", memberNames="Interactive|Batch")
```

Comment:
Should this be another enum, like the one above? `0 | 1`

Reply:
Agreed that consistency would be better, but this schema has to follow the types declared by the plugin. See https://github.com/grafana/google-bigquery-datasource/blob/18680e42ba557791d109c7c540c2c3f2647592f0/src/types.ts#L13-L16

Reply:
Thanks for pointing that out! I agree that consistency would be nice 😅 Oh well.